src/Sylius/Bundle/WebBundle/Resources/views/Frontend/Product/_singleBox.html.twig

```diff
@@ -2,6 +2,9 @@
     <a href="{{ path(product) }}"><h4>{{ product.name|truncate(18) }}</h4></a>
     <a href="{{ path(product) }}">
         <img class="img-rounded img-responsive" src="{{ product.image ? product.image.path|imagine_filter('sylius_medium') : 'http://placehold.it/262x255' }}" alt="{{ product.name }}" />
+        {% if (product.masterVariant.crossedprice != 0) %}
```

Comment:
I think the check here should be if crossedPrice > price, also I think this could be done in a macro as it's like you would want to display the price in this way in lots of places